scripts/gather-package.bat

```diff
@@ -1,13 +1,22 @@
 rem Create empty directories for package bundle
 @echo off
 
+IF "%PLATFORM%"=="x86" (
+    SET FOLDER_PLATFORM="32"
+) ELSE IF "%PLATFORM%"=="x64" (
+    SET FOLDER_PLATFORM="64"
+) ELSE (
+    echo "Platform %PLATFORM% is not supported"
+    exit 1
+)
+
 md %APPVEYOR_BUILD_FOLDER%\package
 md %APPVEYOR_BUILD_FOLDER%\package\include
 md %APPVEYOR_BUILD_FOLDER%\package\include\win
 md %APPVEYOR_BUILD_FOLDER%\package\bin        
 md %APPVEYOR_BUILD_FOLDER%\package\lib
-md %APPVEYOR_BUILD_FOLDER%\package\pthread-win32
-md %APPVEYOR_BUILD_FOLDER%\package\openssl-win64
+rem md %APPVEYOR_BUILD_FOLDER%\package\pthread-win%FOLDER_PLATFORM%
```

Comment:
No use of this commented line.

Reply:
sorry, was a hang-over from changes - removed